ecommerce/frontend/src/components/TrendingGrid.less

```diff
@@ -6,21 +6,57 @@
 }
 
 .gallery__item {
+    position: relative;
     overflow: hidden;
 }
 
+.gallery__item:hover .gallery__img {
+    opacity: 0.4;
+}
+
+.gallery__item:hover .product-info {
+    opacity: 1;
+}
+
 .gallery__img {
+    opacity: 1;
+    display: block;
     width: 100%;
+    transition: .5s ease;
+    backface-visibility: hidden;
```

Comment:
what is backface visibility for? seems like for 3d stuff..

Reply:
Removed.